integration-test/976-fractional-pois.py

```diff
@@ -12,15 +12,15 @@
     'name': 'New Jersey - Pennsylvania' })
 
 assert_has_feature(
-    5, 9, 12, 'roads',
-    { 'min_zoom': 3.0 , 'id': int, 'sort_key': 383,
+    7, 37, 48, 'roads',
+    { 'min_zoom': 3 , 'id': int, 'shield_text': '95',
     'source': 'naturalearthdata.com' })
```

Comment:
I changed the tile here to zoom 7, as we strip some properties from roads when `zoom < 7`, including the `shield_text`, `name` and `ref`. This makes the test quite non-specific, especially as `id` is not reliably constant when coming from shapefile fixtures. At zoom 7, it's still using NaturalEarth data, but the `shield_text` is present so we've more confidence it's matching the feature it was intended to.